build/yamls/flow-aggregator/base/conf/flow-aggregator.conf

```diff
@@ -1,9 +1,13 @@
 # Provide the flow collector address as string with format <IP>:<port>[:<proto>], where proto is tcp or udp.
 # If no L4 transport proto is given, we consider tcp as default.
-flowCollectorAddr: ""
+externalFlowCollectorAddr: ""
```

Comment:
Pasted my comment on another PR here too: we generally comment out the default config option value in yamls.

Reply:
Yes, I saw this in antrea configmap @jianjuns. Since this is working, I did not raise the comment. Wondering why the convention of commenting out the default values is followed?

Reply:
I think just because the default values need not be populated.

Reply:
Thanks jianjun, will comment out these config values.